lib/beaker/host/unix/exec.rb

```diff
@@ -278,20 +278,12 @@ def clear_env_var key
   # @return [Result] result of restarting the SSH service
   def ssh_service_restart
     case self['platform']
-    when /debian|ubuntu/
-      exec(Beaker::Command.new("service ssh restart"))
-    when /amazon|(el|centos|redhat|oracle|scientific)-[7-9]|fedora|archlinux-/
-      exec(Beaker::Command.new("systemctl restart sshd.service"))
-    when /el-|centos|redhat|oracle|scientific/
-      exec(Beaker::Command.new("/sbin/service sshd restart"))
```

Comment:
How about keeping this but make it `/(el|centos|redhat|oracle|scientific)-[0-6]/` so we assume systemd going forward.

Reply:
done